app/src/main/java/com/yterletskyi/happyfriend/features/settings/ui/SeparateLineItem.kt

```diff
@@ -0,0 +1,31 @@
+package com.yterletskyi.happyfriend.features.settings.ui
+
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import androidx.core.view.get
+import androidx.recyclerview.widget.RecyclerView
+
+class SeparateLineItem : RecyclerView.ItemDecoration() {
+
+    private val paint = Paint().apply {
+        color = Color.parseColor("#d3d3d3")
+    }
+
+    override fun onDraw(c: Canvas, parent: RecyclerView, state: RecyclerView.State) {
+        val width = parent[parent.childCount - 1].width.toFloat()
+        val y = parent[parent.childCount - 1].y
+        c.drawLine(
+            0f + separator_line_width,
+            y - separator_line_height,
+            width - separator_line_width,
+            y - separator_line_height,
+            paint
+        )
+    }
+
+    companion object {
+        const val separator_line_height = 8f
```

Comment:
i'm not sure `height` is the best name here. If I make it `80` for example I expect the line to be thicker - but instead it moves down. Maybe a margin would be a better name?